network/p2p/libp2pNode.go

```diff
@@ -62,28 +63,11 @@ func DefaultLibP2PNodeFactory(log zerolog.Logger, me flow.Identifier, address st
 		pubsub.WithMaxMessageSize(maxPubSubMsgSize),
 	}
 
-	ip, port, err := net.SplitHostPort(address)
-	if err != nil {
-		return nil, fmt.Errorf("failed to create middleware: %w", err)
-	}
-
-	// creates libp2p host and node
-	nodeAddress := NodeAddress{Name: me.String(), IP: ip, Port: port}
-
```

Comment:
We no longer need a translation from identifier and address into node address. We pass it directly to `libP2PNode`.